src/nice.bootstrap/_alert.scss

```diff
@@ -3,7 +3,7 @@
 .alert {
   background: $colour-caution-background;
   border: 1px solid $colour-caution;
-  color: $colour-caution;
```

Comment:
@1996dylanriley  we should still use $colour-caution from the design system, but we should darken than (e.g. to #805d25) rather than changing to true black. This way it will affect the border colour too.

Reply:
@ediblecode Here is the PR in NDS https://github.com/nhsevidence/nice-design-system/pull/47